main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ParentStationValidator.java

```diff
@@ -156,4 +186,22 @@ static class WrongParentLocationTypeNotice extends ValidationNotice {
       this.expectedLocationType = expectedLocationType;
     }
   }
+
+  /**
+   * Unused parent station.
+   *
+   * <p>Severity: {@code SeverityLevel.INFO}
+   */
+  static class UnusedParentStationNotice extends ValidationNotice {
+    private final int csvRowNumber;
+    private final String stopId;
+    private final String stopName;
```

Comment:
As part of #1324, now the `ValidationNotice` classes contain their own documentation. We will be deprecating the rules.md soon in favor of inline documentation and GtfsValidationNotice annotation. More information regarding the `new pattern` for adding a notice can be found at [NEW_RULES.md](https://github.com/MobilityData/gtfs-validator/blob/master/docs/NEW_RULES.md)
Find below a suggested change to align this PR with the latest documentation requirements:
```suggestion
  /**
   * Unused parent station.
   *
   * <p>A stop has `location_type` STATION (1) but does not appear in any stop's `parent_station`.
   */
  @GtfsValidationNotice(
          severity = SeverityLevel.INFO,
          files = @FileRefs({GtfsStopSchema.class}))
  static class UnusedParentStationNotice extends ValidationNotice {
    /** The row number of the faulty record. */
    private final int csvRowNumber;

    /** The id of the faulty stop. */
    private final String stopId;

    /** The name of the faulty stop. */
    private final String stopName;
```